service-11/PUS-11-1.acn

```diff
@@ -0,0 +1,28 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-11-1 DEFINITIONS ::= BEGIN
+
+TC-11-1-EnableTheTimebasedScheduleExecutionFunction []
```

Comment:
I always wonder - should we keep words like "the" in names (to be more word-to-word with standard) or should we skip them to make our ids little shorter

Reply:
We might as well leave it as just TC-11-1 and move full name to the comment

Reply:
nah, I just wondered :) leave it as is